label_studio/tasks/migrations/0035_auto_20221221_1116.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.16 on 2022-12-21 11:16
+import logging
+
+from django.db import migrations
+
+logger = logging.getLogger(__name__)
+
+
+def forwards(apps, schema_editor):
+    if not schema_editor.connection.vendor.startswith('postgres'):
+        logger.info('Database vendor: {}'.format(schema_editor.connection.vendor))
+        logger.info('Skipping dropping tasks_annotations_result_idx index')
+        return
+
+    schema_editor.execute('drop index if exists tasks_annotations_result_idx;')
+    schema_editor.execute('drop index if exists tasks_annotations_result_idx2;')
+    schema_editor.execute(
```

Comment:
it might be a long process, more than 10-20 minutes. Shouldn't we move it to async migration?